WalletWasabi/Nostr/NostrCoordinator.cs

```diff
@@ -0,0 +1,34 @@
+using NBitcoin;
+using WalletWasabi.Exceptions;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Nostr;
+
+public record NostrCoordinator(string Description, string Name, Uri Uri, Network Network)
+{
+	private static readonly NostrCoordinator Current = new(
+		Description: "WabiSabi Coinjoin Coordinator",
+		Name: "WabiSabi Coordinator",
+		Uri: new Uri(Constants.BackendUri),
```

Comment:
WTF! I didn't know this was hardcoded. Those hardcoded urls must be removed asap.

All these values must be configurable. I imagine most backend operators would want to announce their servers as "The greatest backend ever".